src/envo/envo-edit.owl

```diff
@@ -41052,7 +41052,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501184> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501185> (dental clinic)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501185> "A medical clinic which is used to provide oral medicine services."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501185> "A medical clinic which is used to provide oral medicine services."@en)
```

Comment:
@ddooley Seems a bit broad/vague. Perhaps: A hospital care unit which is used to provide medical services focused on maintaining and improving the oral health.

Reply:
Need to stick with "medical clinic" as parent since dental clinics aren't necessarily in a hospital setting.  Changed def to: "A medical clinic which is used to provide medical services focused on maintaining and improving a client's oral health."